packages/vanilla/src/sass/base/_core.scss

```diff
@@ -13,6 +13,13 @@ body {
   background-color: var(--cbp-color-branding-cbp-dark);
 }
 
+p,
+blockquote,
+ul:not(ul ul):not(ul ol),
+ol:not(ol ol):not(ol ul) {
+  margin-bottom: var(--cbp-space-4x);
```

Comment:
We need to make a decision on this because we're going back and forth. Is a class really required on a `p` tag to prevent it from running together into a wall of text? This is a super basic style on default HTML tags that I believe is needed as the base level. The rules on the list are just to make sure they have space below them as well, except when nested,